app/initialize.js

```diff
@@ -1,20 +1,20 @@
 import application from './application';
-import { syncFiles } from './libs/file';
-import scdl from './libs/soundcloud';
+import Polyglot from 'node-polyglot';
 
+function initLocale() {
+    let locale = document.documentElement.getAttribute('lang');
```

Comment:
Thanks for adding l18n! We just face an issue here: the static app can't customize the `html[lang]` attribute woth the locale setted in the Cozy, so we must perform a cozysdk request to get it (see https://github.com/cozy/cozy-contacts/blob/master/server/controllers/application.coffee#L20) and:
1. update the `html[lang]` attribute to reflect the document content locale
2. initialize the locale properly